pollux/core/src/main/scala/org/hyperledger/identus/pollux/core/service/CredentialDefinitionService.scala

```diff
@@ -38,22 +38,15 @@ object CredentialDefinitionService {
 
     final case class RepositoryError(cause: Throwable) extends Error
 
-    final case class GuidNotFoundError(guid: UUID) extends Error {
-      def message = s"Credential Definition record cannot be found by `guid`=$guid"
-    }
-    final case class IdNotFoundError(id: UUID) extends Error {
-      def message = s"Credential Definition record cannot be found by `id`=$id"
-    }
+    final case class NotFoundError(guid: Option[UUID] = None, id: Option[UUID] = None, message: String) extends Error
 
-    // final case class NotFoundError(guid: Option[UUID] = None, id: Option[UUID] = None, message: String) extends Error
+    object NotFoundError {
```

Comment:
I guess these case classes are obsolete. We now rely on the new `CredentialDefinitionServiceError` file/trait, right?

Reply:
fixed